### ISSUE TITLE
fatal error caused by systemd/sysvinit confusion on Ubuntu 15.10 (wily).

### ISSUE THREAD

**Issue by kurtlieber** [ansible/ansible-modules-core#2846]
This appears to be the same issue discussed [here](https://github.com/ansible/ansible-modules-core/issues/1563) and [here](https://github.com/ceph/ceph-cm-ansible/pull/49).  Namely, ansible appears to get confused with packages that install both a systemd as well as a sysvinit script.  However, the issue is still present in ansible 1.9.2 and Ubuntu 15.10:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 15.10
Release:        15.10
Codename:       wily

$ ansible --version
ansible 1.9.2
  configured module search path = None
```

ansible was installed from the default Ubuntu repository.

I'm using a very simple playbook, called via a role:

test.yml:

```
- hosts: local
  roles:
    - common
```

main.yml for common:

```
    - name: be sure ntp is installed
      apt: name=ntp state=installed
      tags: ntp

    - name: be sure ntp is configured
      file: src=ntp.conf dest=/etc/ntp.conf
      notify:
        - restart ntpd
      tags: ntp

    - name: be sure ntpd is running and enabled
      service: name=ntpd state=running enabled=yes
      tags: ntp
```

running this produces the following output:

```
$ ansible-playbook -s test.yml

PLAY [local] ******************************************************************

GATHERING FACTS ***************************************************************
ok: [localhost]

TASK: [common | be sure ntp is installed] *************************************
ok: [localhost]

TASK: [common | be sure ntp is configured] ************************************
ok: [localhost]

TASK: [common | be sure ntpd is running and enabled] **************************
failed: [localhost] => {"failed": true}
msg: Error when trying to enable ntpd: rc=1 Failed to execute operation: No such file or directory


FATAL: all hosts have already failed -- aborting

PLAY RECAP ********************************************************************
           to retry, use: --limit @/home/ansible/test.retry

localhost                  : ok=3    changed=0    unreachable=0    failed=1
```

However, NTP does appear to be correctly installed, configured and running after the playbook has run:

```
$ sudo systemctl status ntp
● ntp.service - LSB: Start NTP daemon
   Loaded: loaded (/etc/init.d/ntp)
   Active: active (running) since Tue 2016-01-19 19:08:43 MST; 43min ago
     Docs: man:systemd-sysv-generator(8)
   CGroup: /system.slice/ntp.service
           └─25377 /usr/sbin/ntpd -p /var/run/ntpd.pid -g -u 123:131
```

The ntp package does install an init script as well:

```
$ sudo /etc/init.d/ntp status
● ntp.service - LSB: Start NTP daemon
   Loaded: loaded (/etc/init.d/ntp)
   Active: active (running) since Tue 2016-01-19 19:08:43 MST; 45min ago
     Docs: man:systemd-sysv-generator(8)
   CGroup: /system.slice/ntp.service
           └─25377 /usr/sbin/ntpd -p /var/run/ntpd.pid -g -u 123:131
```

As mentioned in [1563](https://github.com/ansible/ansible-modules-core/issues/1563), this is (was) a known bug, but it was marked as fixed by [#999](https://github.com/ansible/ansible-modules-core/pull/999).  However, it does not appear to be fully fixed, so opening a new issue.  I'm new to ansible, so apologies if I missed something obvious here.

**Comment by bcoca** [comment]
That fix went into the 2.0 version, that means that the pre existing 1.9.2 will still have the bug.

**Comment by kurtlieber** [comment]
> That fix went into the 2.0 version, that means that the pre existing 1.9.2 will still have the bug.

I don't believe that's accurate.  It's a [one-line fix](https://github.com/ansible/ansible-modules-core/pull/999/files) and, when I check my 1.9.2 system/service.py file, it appears to be included:

```
474         # If no service control tool selected yet, try to see if 'service' is available
475         if self.svc_cmd is None and location.get('service', False):
476             self.svc_cmd = location['service']
```
